app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [x] Make sure that you cover all use cases, the application is not asking for confirmation of the password
```suggestion
devise :database_authenticatable, :registerable, :recoverable, 
            :rememberable, :validatable, :confirmable
```